scripts/test.sh

```diff
@@ -58,17 +58,17 @@ do
         ngx new --no-analytics --automate "$CWD/$file" "$TEST_APP_NAME" --no-insights
 
         # cordova android
-        npm run cordova:prepare --no-progress
-        npm run cordova:build android --no-progress
+        npm run cordova:prepare
+        npm run cordova:build android
 
     else
 
         # generators/app test
         ngx new --no-analytics --automate "$CWD/$file" "$TEST_APP_NAME" --no-insights
 
-        npm run test:ci -- --no-progress
-        npm run e2e -- --no-progress
-        npm run build -- --no-progress
+        npm run test:ci --
```

Comment:
`--progress` should only be (temporarily) disabled for e2e tests, after update to latest master the fix is already here

Reply:
awesome. ok.

Reply:
reverted :)